app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -21,6 +21,12 @@ class ManageIQ::Providers::Openstack::CloudManager < ManageIQ::Providers::CloudM
   require_nested :Template
   require_nested :Vm
 
+  has_one :network_manager,
+          :foreign_key => :parent_ems_id,
+          :class_name  => "ManageIQ::Providers::Openstack::NetworkManager",
+          :autosave    => true,
+          :dependent   => :destroy,
+          :inverse_of  => :parent_manager
```

Comment:
Okay some weird rails stuff going on, this is causing the failure somehow:
```
           :class_name  => "ManageIQ::Providers::Openstack::NetworkManager",
           :autosave    => true,
-          :dependent   => :destroy,
-          :inverse_of  => :parent_manager
+          :dependent   => :destroy
   has_many :storage_managers,
```

All green with inverse_of removed